### PR TITLE
#400 Polls

### PR DESCRIPTION
**Pull request by SebinSong**
A follow-up PR for the unresponsive #1628

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-  "contractsVersion": "0.1.4",
+  "contractsVersion": "0.1.5",
   "private": true,
   "description": "",
   "scripts": {
     "backend": "npx nodemon --watch backend/ -- --require ./Gruntfile.js --require @babel/register ./backend/index.js",
     "sass-dev": "node-sass --output-style nested --source-map true -w -r -o ./dist/assets/css ./frontend/assets/style",
     "sass-dist": "node-sass --output-style compressed -o ./dist/assets/css ./frontend/assets/style",
-    "cy:open": "cypress open",
+    "cy:open": "CYPRESS_BASE_URL=http://localhost:8000 cypress open",
```
Comment (SebinSong) [inline package.json:11]: The removal of `baseUrl` field in `cypress.json` in [this](https://github.com/okTurtles/group-income/commit/9bf49a43aa9ad111652ef33d1deb010c551b7b25) recent commit has broken `cypress open` command, so added this env variable.

Comment (taoeffect) [reply package.json:11]: Interesting! Thanks for catching that! /cc @snowteamer

File: frontend/views/containers/chatroom/poll-message-content/PollToVote.vue [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    selectedVotesAsString () {
+      return this.allowMultipleChoices
+        ? this.form.selectedOptions.map(id => this.getOptValue(id)).join(', ')
+        : this.getOptValue(this.form.selectedOptions)
+    }
+  },
+  methods: {
+    submitVotes () {
+      try {
+        sbp('gi.actions/chatroom/voteOnPoll', {
```
Comment (taoeffect) [inline frontend/views/containers/chatroom/poll-message-content/PollToVote.vue]: [Forgot this one!](https://github.com/okTurtles/group-income/pull/1628#discussion_r1271373035)

**Review by taoeffect — APPROVED** [review]
Approved! Great job @SebinSong! 😄 🎉

**Pull request merged**